tools/fs/README_ROMFS.md

```diff
@@ -12,7 +12,7 @@ Select menu.
 File Systems -> ROMFS to y
 ```
 
-2. Put your contents at *external/contents* folder.
+2. Put your contents at *tools/fs/contents* folder.
```

Comment:
At Line No : 2 ( # Contents for ROMFS)

I think now,  tools/fs/contents is generic space for any fs "contents", So do we need to modify accordingly here?